Lib/email/_header_value_parser.py

```diff
@@ -1049,6 +1061,10 @@ def get_encoded_word(value):
         _validate_xtext(vtext)
         ew.append(vtext)
         text = ''.join(remainder)
+    # Encoded words should be followed by a WS
+    if value and value[0] not in WSP:
+        ew.defects.append(errors.InvalidHeaderDefect(
+            "missing trailing whitespace after encoded-word"))
```

Comment:
Since this same defect can occur when the whitespace is missing both before and after encoded word, and does not distinguish between these two cases (which is fine IMHO), what about the following for the message?

> encoded-word is not deliminted by whitespace

Reply:
Actually, it looks like right error messages are produced based on the whitespace was missing before or after the encoded word. It is just not in this diff, hence I missed that when we were reviewing.
Running the tests and printing the actual exceptions produces correct exceptions:
```
InvalidHeaderDefect('missing whitespace before encoded word')
&
InvalidHeaderDefect('missing trailing whitespace after encoded-word')
```